ui/src/app/views/quick-tour-modal/component.tsx

```diff
@@ -0,0 +1,222 @@
+        you can go through and manually review the records for each participant and decide if
+        you want to include or exclude them from your Cohort and make specific
+        annotations/notes to each record.</div>,
+      image: '/assets/images/cohorts.png'
+    },
+    {
+      title: 'Concepts',
+      content: <div>Concepts describe information in a patient’s medical record, such as a condition
+          they have, a  prescription they are taking or their physical measurements. In the Workbench
+          we refer to subject areas such as conditions, drugs, measurements etc. as “domains.” You can
+          search for and save collections of concepts from a particular domain as a “Concept Set.”
+          <br/><br/>
+          For example, if you want to select height, weight and blood pressure information
+          (concepts) from your “diabetes cases” Cohort, you can search for the 3 concepts
+          from the “Measurements” domain and call it “biometrics” Concept Set. You can then
+          use Notebooks to extract that information from your cohort.</div>,
+      image: '/assets/images/concepts.png'
+    },
+    {
+      title: 'Notebooks',
+      content: <div>A Notebook is a computational environment where you can analyze data with basic
+          programming knowledge in R or Python. Several template Notebooks and resources
+          are available within your Workspace that will guide you how to import your
+          Cohort(s) and Concept Set(s) into the Notebook and can assist with basic analyses.
+          <br/><br/>
+         For example, you can launch a Notebook
+          to import your “diabetes cases” Cohort and then select your “biometrics” Concept Set, to
+          get biometrics data for the participants in your Cohort. You can then analyze the data to
+          study correlation between hypertension and diabetes.</div>,
+      image: '/assets/images/notebooks.png'
+    }];
+
+class QuickTourReact extends React.Component<any, any> {
+  state: {selected: number, fullImage: boolean, numPanels: number};
+  props: {learning: boolean, closeFunction: Function};
+
+  checkImg = '/assets/images/check.svg';
+  expandIcon = '/assets/icons/expand.svg';
+
+  constructor(props: Object) {
+    super(props);
+    this.state = {selected: 0, fullImage: false, numPanels: 5};
+  }
+
+  previous(): void {
+    this.setState({selected: this.state.selected - 1});
+  }
+
+  next(): void {
+    if (this.state.selected === 4) {
+      this.close();
+      return;
+    }
+    this.setState({selected: this.state.selected + 1});
+  }
+
+  close(): void {
+    this.setState({selected: 0});
+    this.props.closeFunction();
+  }
+
+  selectPanel(panel: number): void {
+    this.setState({selected: panel});
+  }
+
+  lastButtonText(): string {
+    return (this.state.selected === 4 ? 'Close' : 'Next');
+  }
+
+  toggleImage(): void {
+    this.setState({fullImage: !this.state.fullImage});
+  }
+
+  render() {
+      return  <>
```

Comment:
What is this for? I haven't seen it in other React code (I haven't looked at much)

Reply:
again, Brett will know better than me, but it was some mechanism for returning multiple divs that are not all children of each other

Reply:
from brett: 

> The empty tag is called a Fragment, and it's basically a way to bundle up several DOM nodes together without wrapping them in an actual element. It's nice so we don't end up with extraneous wrappers in the DOM